packages/design-system-twrnc-preset/package.json

```diff
@@ -46,14 +46,15 @@
     "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch"
   },
   "dependencies": {
-    "@metamask/design-tokens": "workspace:^"
+    "twrnc": "^4.5.1"
   },
   "devDependencies": {
     "@babel/core": "^7.23.5",
     "@babel/preset-env": "^7.26.0",
     "@babel/preset-react": "^7.25.9",
     "@babel/preset-typescript": "^7.23.3",
     "@metamask/auto-changelog": "^5.0.2",
+    "@metamask/design-tokens": "workspace:^",
```

Comment:
Adding "@metamask/design-tokens": "workspace:^" to devDependencies ensures the package can still be built successfully, even after moving it from dependencies to peerDependencies.